packages/javascript/bh-shared-ui/src/commonSearches.tsx

```diff
@@ -210,6 +210,34 @@ export const CommonSearches: CommonSearchType[] = [
                 description: 'Enabled Tier Zero / High Value principals inactive for 60 days',
                 cypher: `WITH 60 as inactive_days\nMATCH (n)\nWHERE n.system_tags CONTAINS "admin_tier_0"\nAND n.enabled = true\nAND n.lastlogontimestamp < (datetime().epochseconds - (inactive_days * 86400)) // Replicated value\nAND n.lastlogon < (datetime().epochseconds - (inactive_days * 86400)) // Non-replicated value\nAND n.whencreated < (datetime().epochseconds - (inactive_days * 86400)) // Exclude recently created principals\nAND NOT n.name STARTS WITH "AZUREADKERBEROS." // Removes false positive, Azure KRBTGT\nAND NOT n.objectid ENDS WITH "-500" // Removes false positive, built-in Administrator\nAND NOT n.name STARTS WITH "AZUREADSSOACC." // Removes false positive, Entra Seamless SSO\nRETURN n`,
             },
+            {
```

Comment:
There was an update to set limits on all of these prebuilt queries recently. Would it be good to put limits on these new queries as well?

Reply:
Good call, will update 👍